### PR TITLE
webui: item reforge

### PR DESCRIPTION
**Pull request by or2e**

### CODE REVIEW

File: src/WebUI/locales/en.yml [hunk 1]
```diff
@@ -502,13 +503,31 @@ character:
         notify:
           warning: A large shield cannot be used on horseback
       upgrade:
-        upgradesTitle: Item Upgrades
+        upgradesTitle: "Item upgrades"
+        tooltip:
+          title: "Upgrade item"
+          description: "You can upgrade your item for a heirloom point"
```
Comment (Yeldur) [inline src/WebUI/locales/en.yml:509]: Recommend change to -> "You can upgrade your item by spending one heirloom point"

File: src/WebUI/locales/en.yml [hunk 2]
```diff
@@ -502,13 +503,31 @@ character:
...
-          description: 'Are you sure you want to upgrade this item for {loomPoints}? Your item {oldItem} will be changed to {newItem}'
-          value: Upgrade item
+          description: "Are you sure you want to upgrade this item for {loomPoints}? Your item {oldItem} will be changed to {newItem}"
         currentItem: Current item
         notify:
           success: Item successfully upgraded
+      reforge:
+        tooltip:
+          title: "Reforge item"
+          description: "You can reforge your items for a gold"
```
Comment (Yeldur) [inline src/WebUI/locales/en.yml:522]: Recommend change to -> "You can reforge your items for gold"

File: src/WebUI/locales/en.yml [hunk 3]
```diff
@@ -502,13 +503,31 @@ character:
         notify:
           warning: A large shield cannot be used on horseback
       upgrade:
-        upgradesTitle: Item Upgrades
+        upgradesTitle: "Item upgrades"
+        tooltip:
+          title: "Upgrade item"
+          description: "You can upgrade your item for a heirloom point"
+        validation:
+          maxRank: "The item already has a maximum rank"
```
Comment (Yeldur) [inline src/WebUI/locales/en.yml:511]: Recommend change to -> "This item has already been upgraded to the highest point possible"

File: src/WebUI/locales/en.yml [hunk 4]
```diff
@@ -502,13 +503,31 @@ character:
...
+          loomPoints: "You don't have enough heirloom points"
+          exist: "You already have an improved version of this item"
         confirm:
-          description: 'Are you sure you want to upgrade this item for {loomPoints}? Your item {oldItem} will be changed to {newItem}'
-          value: Upgrade item
+          description: "Are you sure you want to upgrade this item for {loomPoints}? Your item {oldItem} will be changed to {newItem}"
```
Comment (Yeldur) [inline src/WebUI/locales/en.yml:515]: Would this read as "Are you sure you want to upgrade this item for **{1}**? Your item **{+1 hoe}** will be changed to **{+2 hoe}**"? 
If yes:
Recommend change to -> Are you sure you want to upgrade this item for **{loomPoints} loompoint?** Your item **{oldItem}** will be changed to **{newItem}**

File: src/WebUI/locales/en.yml [hunk 5]
```diff
@@ -502,13 +503,31 @@ character:
...
         currentItem: Current item
         notify:
           success: Item successfully upgraded
+      reforge:
+        tooltip:
+          title: "Reforge item"
+          description: "You can reforge your items for a gold"
+        validation:
+          rank: "The item must have a rank greater than {minimumRank}"
+          gold: "You don't have enough gold"
```
Comment (Yeldur) [inline src/WebUI/locales/en.yml:525]: Recommend change to -> "You don't have enough gold to perform this action"
OR
"Not enough gold to perform this action"

File: src/WebUI/locales/en.yml [hunk 6]
```diff
@@ -502,13 +503,31 @@ character:
...
+          exist: "You already have the {existItem} in your inventory. Sell it first"
+        confirm:
+          description: "Are you sure you want to reforge this item for {gold}? You will get {loomPoints}, also item {oldItem} will be changed to {newItem}"
```
Comment (Yeldur) [inline src/WebUI/locales/en.yml:528]: Would this read as "Are you sure you want to reforge this item for {gold}? You will get **{1}**, also **{+1 hoe}** will be changed to **{hoe}**"?

If yes:
Recommend change to -> "Are you sure you want to reforge this item for {gold}? You will get {loomPoints} loompoint back, in exchange {oldItem} will be changed to {newItem}"

else:
Recommend change to -> "Are you sure you want to reforge this item for {gold}? You will get {loomPoints} back, **in exchange** {oldItem} will be changed to {newItem}"

**Review by Yeldur — CHANGES_REQUESTED** [review]
Recommended grammatical changes, general fluff and queries on existing items dependant on how they would read.

**Review by verdie-g — APPROVED** [review]

**Pull request merged**